### PR TITLE
Redesign homepage

### PR DESCRIPTION
**Pull request by aronstrandberg**
I will create several smaller PRs into this one before merging the entire redesign into master to make reviewing easier

- [x] Update title
- [ ] Update favicon
- [ ] Update theme colors in manifest.json

### CODE REVIEW

**Comment by jeppes** [comment]
👍

File: app/translations/se.json [hunk 1]
```diff
@@ -1,47 +1,52 @@
...
 }, {
-	"id": "app.components.HomePage.contact.content",
-	"message": "Är ni intresserade av att ha ett event med oss eller bara är nyfikna och vill veta mer, kontakta vår säljansvarig Elin Karagöz."
+  "id": "app.components.HomePage.trip.header",
+  "message": "Resan"
 }, {
-	"id": "app.components.HomePage.contact.phone",
-	"message": "Telefon"
+  "id": "app.components.HomePage.trip.content",
+  "message": "Projektet avslutas med en resa till USA, för att skapa kulturellt och professionellt utbyte. MEd städer som San Francicsco, New York och Seatttle har USA en unik position i IT-världen med ständig innovation från både startups och etablerade spelare."
```
Comment (jeppes) [inline app/translations/se.json]: MEd --> Med

File: app/translations/en.json [hunk 2]
```diff
@@ -5,23 +5,32 @@
...
+  "id": "app.components.HomePage.events.header",
+  "message": "The Events"
+}, {
+  "id": "app.components.HomePage.events.content",
+  "message": "Throughout the year, the Studs members will visit the companies through events, usually arranged at their offices. The students take this opportunity to gain deeper insight into life at the company and its operations. Studs helps the companies tailor the events to their specific needs, guaranteeing each event is as unique as the company hosting it."
```
Comment (jeppes) [inline app/translations/en.json]: visit the companies through events, usually arranged at their offices --> visit various companies at events usually held at their offices.
and its operations --> and their operations
Studs helps the companies --> Studs helps these companies
guaranteeing each event --> guaranteeing that each event

**Comment by jeppes** [comment]
Just a few things, looking really good so far though! 💯 🥇

**Comment by aronstrandberg** [comment]
@jeppes Fixed your comments and changed to the copy by Informationsgruppen

**Pull request merged**